install_clam.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+apt-get update -qq
+apt-get install -qq clamav-daemon
+echo TCPSocket 3310 >> /etc/clamav/clamd.conf
+cat /etc/clamav/clamd.conf
```

Comment:
Was for debugging but I figured its useful should it suddenly start breaking due to an updated clam package or something